### PR TITLE
Added support for inline SVG as source

### PR DESCRIPTION
**Pull request by KartikSoneji**
#2 I have added inline SVG support as the source element in Layer. Please have a look.

### CODE REVIEW

File: src/utils/getElement.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ export const getAsyncElement = (...params) => {
   const element = resolveValidElement(...params);
 
   if (element instanceof HTMLImageElement) return imageLoader(element);
-  if (element instanceof HTMLCanvasElement) return Promise.resolve(getElementInfo(element));
+  else if (element instanceof HTMLCanvasElement) return Promise.resolve(getElementInfo(element));
```
Comment (hbmueller) [inline src/utils/getElement.js]: You don't need to add the `else` here, since it is using early returns. This was a code style option from my part.

**Review by hbmueller — CHANGES_REQUESTED** [review]
Sorry for taking this long to review this PR.

Did a quick check and the code doesn't seem to be working. Some additional changes might be necessary. Also, `createElement` has not been imported in this file.

File: src/utils/getElement.js (hunk 1 above)
Comment (KartikSoneji) [reply src/utils/getElement.js]: I have fixed the formatting. Personally, I prefer to have the `else if` as it serves as a visual indicator that only one of the if statements should run.

**Comment by KartikSoneji** [comment]
I have made the necessary changes.
Turns out, Chrome 72 changed the way image src is handled. I have fixed the code to work with the new rules. [More info](https://stackoverflow.com/questions/30733607).

**Pull request closed without merging**